libcontainer/integration/exec_test.go

```diff
@@ -605,9 +605,6 @@ func TestPidsSystemd(t *testing.T) {
 }
 
 func testPids(t *testing.T, systemd bool) {
-	if testing.Short() {
```

Comment:
Why remove this?

Reply:
Just remove `-short` from https://github.com/opencontainers/runc/blob/3cb1909c70f16c36fcbc88f48fab7907eed7b1fc/.travis.yml#L49

Reply:
TESTFLAGS=-short'  let about  53 test skip. But I just solved  _`TestPidsSystemd`_ and `_TestRunWithKernelMemorySystemd_` test cases error.

Reply:
IIUC we no longer need to skip any unit test on CentOS7 CI?

Reply:
OK, I'll have a try.

Reply:
All tests passed, :+1: